src/hotspot/share/opto/loopnode.hpp

```diff
@@ -1739,6 +1739,8 @@ class PhaseIdealLoop : public PhaseTransform {
   void update_addp_chain_base(Node* x, Node* old_base, Node* new_base);
 
   bool can_move_to_inner_loop(Node* n, LoopNode* n_loop, Node* x);
+
+  void pin_array_access_nodes(Node* ctrl);
```

Comment:
```suggestion
  void pin_array_access_nodes_dependent_on(Node* ctrl);
```
Does this even build without?